acme_diags/plot/cartopy/qbo_plot.py

```diff
@@ -69,19 +71,23 @@ def plot(period_new, parameter, test, ref):
     z = dict(data=test['qbo'].T)
     ax0 = plot_panel(0, fig, 'contourf', label_size, '{} U 5S-5N'.format(test['name']), x, y, z=z,
                plot_colors=cmap2, color_levels=color_levels0, color_ticks=[-50, -25, -5, 5, 25, 50])
+    ax0.invert_yaxis()
```

Comment:
I'm not quite sure why we're inverting these axes. Looking at the graphs, pressure seems to be increasing from bottom to top (which would match with your `unify_plev` function); however, since pressure decreases with altitude, I thought the convention was to have pressure decrease going up the y axis? Or am I misinterpreting?

Reply:
The convention showing the vertical profiles is to have pressure decreases going up the y axis (i.e. altitude increases with y). I have pressure decreasing from bottom to top https://acme-viewer.llnl.gov/zhang40/qbo_diags_acme1/viewer/qbo/bvariable/bqbo-era5/bplot.html
Would you post your testing results?

Reply:
Thank you for catching this! It took some time to troubleshooting. It turns out that, the 3.1.0 version of matplotlib was causing the trouble. After I upgrade to 3.1.2. I don't need to use `ax.invert_yaxis()` to see the correct pressure levels. I deleted these three lines in the new commit.